docs/mesos/README.md

```diff
@@ -6,78 +6,63 @@
+Mesos supports several different networking interfaces depending on which
+Containerizer is being used. Calico provides compatible plugins for each.
+Below, we list three guides for each networking interface:
 
-### Unified Containerizer
-Calico with the Unified Containerizer uses the [Calico Mesos plugin]
-(https://github.com/projectcalico/calico-mesos) to configure
-networking for a Mesos agent that is using the [net-modules network
-isolator](https://github.com/mesosphere/net-modules). Networks are
-specified as net-groups when launching a Mesos task.  A Calico
-profile is automatically created for each net-group (if it doesn't
-already exist), defaulting to allow communication between containers
-using the same net-group.  Fine grained policy can be modified using
-the `calicoctl profile` commands.
+1. Quickstart scripts to launch a demo Mesos cluster pre-configured with Calico.
+2. Manual instructions on adding Calico to a standard Mesos Cluster.
+3. Usage guide detailing how to launch applications networked with Calico for
+the described networking interface.
 
-## Cluster Configuration Requirements
-The installation requirements to use Calico networking are different
-depending on whether you are using the Docker Containerizer or the
-Unified Containerizer.  When setting up your cluster, follow the
-appropriate guide based on your requirements.
+### a.) Docker Containerizer
+Tasks launched in Mesos using the Docker Containerizer (i.e. Docker Engine) are
+networked by Calico's Docker-Libnetwork plugin. Once installed on each Mesos
+Agent, operators can create Docker networks up-front, then launch Docker
+tasks on them.
 
-Calico is particularly suitable for large Mesos deployments on bare
-metal or private clouds, where the performance and complexity costs of
-overlay networks can become significant. It can also be used in public
-clouds.
+- [Vagrant Install: Calico for Docker Tasks in Mesos](Vagrant.md)
+- [Manual Install: Calico for Docker Tasks in Mesos](ManualInstallCalicoDockerContainerizer.md)
+- [Usage Guide: Launching Docker Tasks networked by Calico in Mesos](UsageGuideDockerContainerizer.md)
 
-If you have an existing Mesos cluster, follow the appropriate
-installation guide. However, please ensure that Mesos (and Marathon
-if you are using it) are installed at the appropriate minimum
-version, upgrading if necessary.
+### b.) Unified Containerizer with CNI
+Mesos v1.0.0 has introduced first-class support of the [Container Network
+Interface (CNI)](https://github.com/containernetworking/cni) for the Unified
+Containerizer.
 
-## Guides
+- [Quickstart with Docker-Compose: Calico for Mesos CNI](cni-compose-demo/)
+- [Manual Install: Calico for Mesos Tasks (CNI)](ManualInstallCalicoUnifiedContainerizer.md)
+- [Usage Guide: Launching Mesos Tasks networked with Calico CNI](#)
 
-To build a new Mesos cluster with Calico networking, try one of the
-following guides:
+### c.) Unified Containerizer with Net-Modules [Deprecated]
```

Comment:
Since you indicate this as deprecated in the title - perhaps it's worth a few words in the body to  say why.